avm/ptn/app-service-lza/hosting-environment/modules/supporting-services/deploy.supporting-services.bicep

```diff
@@ -0,0 +1,63 @@
+targetScope = 'resourceGroup'
+import { NamingOutput } from '../naming/naming.module.bicep'
+
+// ------------------
+//    PARAMETERS
+// ------------------
+
+param naming NamingOutput
+
+@description('The location where the resources will be created. This needs to be the same region as the spoke.')
+param location string = resourceGroup().location
+
+@description('Optional. The tags to be assigned to the created resources.')
+param tags object = {}
+
+@description('Required. Whether to enable deployment telemetry.')
+param enableTelemetry bool
+
+// Hub
+@description('The resource ID of the existing hub virtual network.')
+param hubVNetResourceId string
+
+// Spoke
+@description('The resource ID of the existing spoke virtual network to which the private endpoint will be connected.')
+param spokeVNetResourceId string
+
+@description('The name of the existing subnet in the spoke virtual to which the private endpoint will be connected.')
+param spokePrivateEndpointSubnetName string
+
+@description('Optional. Resource ID of the diagnostic log analytics workspace. If left empty, no diagnostics settings will be defined.')
+param logAnalyticsWorspaceResourceId string = ''
```

Comment:
```bicep
param logAnalyticsWorkspaceResourceId string = ''
```
Small typo. Recommend a  'search and replace' to catch all other places this may be incorrect too.